e2e/deployers/appset.go

```diff
@@ -58,11 +59,16 @@ func (a ApplicationSet) Undeploy(ctx types.Context) error {
 
 	clusterName, err := util.GetCurrentCluster(util.Ctx.Hub, managementNamespace, name)
 	if err != nil {
-		return err
-	}
+		if !k8serrors.IsNotFound(err) {
+			return err
+		}
 
-	log.Infof("Undeploying applicationset app \"%s/%s\" in cluster %q",
-		ctx.AppNamespace(), ctx.Workload().GetAppName(), clusterName)
+		log.Debugf("Could not retrieve the cluster name, required resource not found: %s", err)
```

Comment:
Thanks for sharing the log:

> DEBUG   subscr-deploy-rbd-busybox       dractions/actions.go:110        Could not retrieve the cluster name, required resource not found: placements.cluster.open-cluster-management.io "subscr-deploy-rbd-busybox" not found

We can remove the ", required resource not found", because the error explains itself. We will have:

```console
DEBUG   subscr-deploy-rbd-busybox       dractions/actions.go:110        Could not retrieve the cluster name: placements.cluster.open-cluster-management.io "subscr-deploy-rbd-busybox" not found
```